avaframe/in1Data/getInput.py

```diff
@@ -228,6 +248,8 @@ def getAndCheckInputFiles(inputDir, folder, inputType):
         subfolder name where the shape file should be located (SECREL, ENT or RES)
     inputType : str
         type of input (used for the logging messages). Secondary release or Entrainment or Resistance
+    fileExt: str
+        file extension e.g. shp, asc - optional default is shp
```

Comment:
At the moment it is not set to a default value. If nothing is provided fileext is none. 
I think it should be optional as mentioned therefore line 238 needs fileExt='shp'
Furthermore: all `shp` infos above (when calling this function) can be removed

Reply:
made it optioanl and default to shp